src/ContainerList.react.js

```diff
@@ -9,8 +9,14 @@ var ContainerList = React.createClass({
   render: function () {
     var self = this;
     var containers = this.props.containers.map(function (container) {
+      var container_id = container.Id;
```

Comment:
Also would you consider something like [jscs](http://jscs.info/) to automatically check those rules? (I've already added `requireCamelCaseOrUpperCaseIdentifiers` to my local `.jscsrc` ;)